src/build/index.js

```diff
@@ -19,13 +23,17 @@ module.exports = async () => {
     // Register the commands with Discord
     const discordCommands = await registerCommands(commands);
 
-    // Export the Discord commands as JSON
-    // Mapped to be keyed by their API ID
+    /*
+     * Export the Discord commands as JSON
+     * Mapped to be keyed by their API ID
+     */
     const discordCommandsObj = discordCommands.reduce((obj, cmd) => {
         obj[cmd.id] = cmd;
         return obj;
     }, {});
-    await fs.writeFile(path.join(__dirname, 'data', 'commands.json'), JSON.stringify(discordCommandsObj, null, 2));
+
+    await fs.writeFile(path
+        .join(__dirname, 'data', 'commands.json'), JSON.stringify(discordCommandsObj, null, 2));
```

Comment:
This seems like an incredibly weird way to wrap a line, why not wrap at the comma?

Reply:
Since we have changed the max-len, i'm just gonna make it one line only, since this and most of the odd wrappings were because of the max-len being set to 105